### ISSUE TITLE
Issue 2: Preprocesamiento del dataset (normalización y one-hot encoding)

### ISSUE THREAD

**Issue by sharafayyoub** [sharafayyoub/Actividad-redes-concolucionales#3]
Descripción:
Antes de alimentar las imágenes a la CNN, debes preprocesarlas para que el modelo entienda correctamente los datos. Esto implica escalar los valores de los píxeles (que van de 0 a 255) al rango [0, 1], y convertir las etiquetas numéricas (0 a 9) en vectores binarios mediante one-hot encoding.

Tareas:

Normalizar las imágenes:

x_train = x_train.astype('float32') / 255.0
x_test = x_test.astype('float32') / 255.0

Convertir las etiquetas:

from tensorflow.keras.utils import to_categorical
y_train = to_categorical(y_train, 10)
y_test = to_categorical(y_test, 10)

Verificar que las formas y los valores sean correctos (y_train[0] debería ser un vector con 10 posiciones).

Criterio de éxito:
Datos listos para usar en una CNN, con imágenes normalizadas en flotantes y etiquetas codificadas correctamente.
